docs/projects/tad/reporting-standard/0.1a4.md

```diff
@@ -0,0 +1,460 @@
+# :label: 0.1a4 (Latest)
```

Comment:
This new file needs to be manually linked in the menu (https://github.com/MinBZK/ai-validation/blob/main/mkdocs.yml#L88)